draft-packages/modal/KaizenDraft/Modal/Presets/InputEditModal.scss

```diff
@@ -11,13 +12,24 @@
 
 .header {
   color: $color-purple-800;
-  padding: $ca-grid $ca-grid $ca-grid * 1.5;
-  text-align: center;
-  border-bottom: 1px solid $ca-border-color;
+  text-align: left;
+  &.textAlignRTL {
+    text-align: right;
+  }
+  &.padded {
+    padding: $ca-grid $ca-grid * 1.5;
+    @include ca-media-mobile {
+      padding: $ca-grid;
+    }
```

Comment:
Nit/one for @ActuallyACat: are we strict on/not leaving a new line before the next declaration?

Reply:
Not really. Out of habit, I add a newline but it's not a codified standard. If it's something we feel strongly about we can create a Stylelint rule.